x-pack/plugins/cloud_security_posture/public/pages/findings/__mocks__/findings.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CspFinding } from '../../../../common/schemas/csp_finding';
+
+export const mockFindingsHit: CspFinding = {
+  result: {
+    evaluation: 'passed',
+    evidence: {
+      serviceAccounts: [],
+      serviceAccount: [],
+    },
+    // TODO: wrong type
+    // expected: null,
```

Comment:
generated with cloudbeat `84e1dea943da96dbfbfa391ca6f13e32d7d79ddf` 

seems like our `CspFinding` interface is outdated